package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "prebid.js",
-      "version": "7.48.0-pre",
+      "version": "7.51.0-pre",
```

Comment:
I suggest leaving this change out of the PR since all it's likely to do is generate merge conflicts.

Reply:
This file sneaked in the commit, I'll change it back to previous value